server/journeys/screen/check-answers/controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, RequestHandler, Response } from 'express'
+import { BaseJourneyController } from '../../base/controller'
+
+export class ScreenCheckAnswersController extends BaseJourneyController {
```

Comment:
it's not making use of anything in `BaseJourneyController`
i guess the intent is to either add createScreening function to the base class, or to expose its csipApiService to be used by the child class?

Reply:
Good catch, it will be in use in the next ticket / confirmation for actually sending the data off to the API